dimod/quadratic/quadratic_model.py

```diff
@@ -1226,6 +1226,25 @@ def vartype(self, v: Variable) -> Vartype:
         """The variable type of the given variable."""
         return self.data.vartype
 
+    def vartype_info(self, vartype: VartypeLike):
```

Comment:
Overall the code works as intended. I'm not sure if there is a clean way to define the method downstream such that it doesn't have to live in `QuadraticModel`. But maybe we can use the `@staticmethod` decorator here so that we don't have to instantiate a QM just to check on variable type information.

Reply:
The way it is currently implemented, I can do:
```
import dimod
dimod.Integer('i').vartype_info(dimod.INTEGER)
```
Which feels equally as clunky as the original issue, although I get a lot more information packaged together. 
The behavior is quite different as what is demonstrated in the issue itself, since the `lower_bound` and `upper_bound` methods would actually return the set bounds on the integer variable, where `vartype_info` gives static information that doesn't depend on an instantiated variable.

It also feels a little clunky that I can use an instantiated QM, and by extension an instantiated `Integer` to check vartype info, but the same is not true for BQMs/Spin/Binary instances.